backend/src/main/resources/db/migration/V1_0_5__Add_Organization_Employees.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE organization CASCADE;
+
```

Comment:
This script can be merged into the previous one

Reply:
Yes, it will be merged soon. 
The reason to have multiple scripts is to not have troubles with the flyway schema history, that does not allow to execute the application if there is changes in previous scripts.
However, now I am wondering if Jenkins does not persist the flyway_schema_history table between builds

Reply:
Our app uses an in-memory DB while running the tests every build so all the scripts are applied from scratch.
The problem could come only between deploys but since this feature is still under development and has not been merged to develop branch it should be fine. You might face issues though if you run your application locally with the docker-compose POSTGRES image which you can easily fix by removing the volume it uses.